huntsman/camera/pyro.py

```diff
@@ -598,18 +650,18 @@ def get_uid(self):
         """
         return self._camera.uid
 
-    def take_exposure(self, seconds, base_name, dark, dir_name=None, *args,
-                      **kwargs):
-        
-        #Specify the full filename
-        #This uses the camera server's "images" directory 
-        if dir_name is None:
-            filename = os.path.join(os.path.abspath(
-                        self.config['directories']['images']), base_name)
-        #This does not necessarily use the "images" directory
-        else:
-            filename = os.path.join(dir_name, base_name)
-                    
+    def take_exposure(self, seconds, filename, dark, *args, **kwargs):
+        '''
+        Parameters
+        ----------
+        filename (str):
+            The filename of the exposure result.
+            
+        Returns
+        -------
+        str:
+            The full filename of the exposure output.
+        '''                    
```

Comment:
Returning the filename is another hangover from the rsync stuff. Doesn't do any harm, but equally we don't need it any more. Would be more helpful to instead return any exceptions raised by the `self._camera.take_exposure()` call (after serializing them first). That doesn't need to be part of this PR, though.